modules/sdk/gradle-plugins-workspace/src/gradleTest/targetPlatformProvidedModulesResolve/build.gradle

```diff
@@ -1,3 +1,11 @@
+allprojects {
```

Comment:
Please follow [these guidelines](https://github.com/liferay/liferay-portal/blob/master/modules/README.markdown#gradle-files), `task ...` should come before `allprojects`.